.vscode/extensions.json

```
@@ -27,6 +27,9 @@
     // Firebase
     // https://marketplace.visualstudio.com/items?itemName=toba.vsfire
     "toba.vsfire",
+
+    // We use Markdown All in One to generate the table of contents.
```

Match the comment format of all the other extensions.

```suggestion
    // Markdown All in One
    // https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one
```